gen/calc.py

```diff
@@ -1128,6 +1128,7 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'mesos_log_retention_count': calculate_mesos_log_retention_count,
         'mesos_log_directory_max_files': calculate_mesos_log_directory_max_files,
         'dcos_version': DCOS_VERSION,
+        'dcos_flavor': 'open',
```

Comment:
+1 Please see if a test can be added to this.

Reply:
I'll definitely add a test, and will also need to update the docs. This is still very much a WIP.

Reply:
Added a test in another commit.

Reply:
cc @Poltergeist

Reply:
Who's the PM for this?  They should fix the name.

Reply:
PM passing through.....Can we use: dcos_distribution?